src/pages/collections/[collection].tsx

```diff
@@ -214,6 +214,10 @@ const CollectionShow: NextPage<CollectionPageProps> = ({
   const { publicKey, connected } = useWallet()
   const [hasMore, setHasMore] = useState(true)
   const router = useRouter()
+  let hostname
+  if (typeof window !== 'undefined') {
+    hostname = window.location.hostname;
```

Comment:
bsed on this stack overflow we can get the host in getServerSideProps from the header.

```
context.req.headers.host
```

https://stackoverflow.com/questions/65199051/how-to-get-page-url-or-hostname-in-nextjs-project

Can you look to pass the hostname as a page prop?

Reply:
@kespinola Done!